packages/Breadcrumbs/src/components/Link/Link.styles.js

```diff
@@ -1,10 +1,8 @@
 import styled from "styled-components";
 import tokens from "@paprika/tokens";
-import { lineHeight, fontSize } from "@paprika/stylers/lib/helpers";
-import Button from "@paprika/button";
-import ArrowLeftIcon from "@paprika/icon/lib/ArrowLeft";
+import { lineHeight } from "@paprika/stylers/lib/helpers";
 
-export const Link = styled(Button.Link)`
+export const Link = styled.a`
   color: ${({ isDark }) => (isDark ? tokens.color.white : tokens.textColor.subtle)};
   display: inline;
```

Comment:
Are all of these styles still necessary since it is only an `a` now?

Reply:
They are still necessary, it's still rendering a `Button.Link`, it's just I moved the default to `Button.Link` logic into the `Link.js` file instead of mixing between `Link.js` and `Link.styles.js`